WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.kt

```diff
@@ -324,8 +324,64 @@ class MySitesPage {
                         ViewMatchers.withText(R.string.dashboard_card_domain_sub_title),
                         ViewMatchers.withId(R.id.dashboard_card_domain_sub_title),
                     )
+                )
+            )
+        )
+            .check(ViewAssertions.matches(ViewMatchers.isCompletelyDisplayed()))
+
+        return this
+    }
+
+    // "Pages" Dashboard Card
+
+    fun scrollToPagesCard(): MySitesPage {
+        return scrollToCard(R.id.dashboard_card_pages)
+    }
+
+    fun tapPagesCard(): PagesScreen {
+        tapCard(R.id.dashboard_card_pages)
+        return PagesScreen()
+    }
+
+    fun assertPagesCard(): MySitesPage {
+        Espresso.onView(
+            Matchers.allOf(
+                ViewMatchers.withId(R.id.dashboard_card_pages),
+                ViewMatchers.isDescendantOfA(ViewMatchers.withId(R.id.dashboard_cards)),
+
+                ViewMatchers.hasDescendant(
+                    Matchers.allOf(
+                        ViewMatchers.withText(R.string.dashboard_pages_card_title),
+                        ViewMatchers.withId(R.id.my_site_card_toolbar_title),
```

Comment:
```suggestion
                        ViewMatchers.withId(R.id.my_site_card_toolbar_title)
```
i noticed that for multiple matches, the last matcher ends with a `,` which looks odd to me since it's the last one. is this intentional or is it a kotlin thing?

Reply:
In this particular case, this is most probably a result of me moving things around. Such a comma in the end, even if it's not followed by another line, is not considered a code style violation. I saw cases when people deliberately kept commas at the end of a line, to avoid extra noise in the future PRs when they add a next line (I left once a nitpick about this [here](https://github.com/woocommerce/woocommerce-ios/pull/9442#discussion_r1165467816) in a PR of yours).